Firestore/core/src/remote/serializer.cc

```diff
@@ -1115,9 +1117,10 @@ OrderBy Serializer::DecodeOrderBy(
   return OrderBy(std::move(field_path), direction);
 }
 
-google_firestore_v1_Cursor Serializer::EncodeBound(const Bound& bound) const {
+google_firestore_v1_Cursor Serializer::EncodeBound(const Bound& bound,
```

Comment:
I am not a huge fan of `as_start` as it is not 100% self explanatory. Can we pass in  `position` and the desired value of `before` instead? FWIW I do think keeping this method is better than inlining it like we did on Android.